packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -482,6 +575,41 @@ export default class Meeting extends StatelessSparkPlugin {
       });
   }
 
+  /**
+   * acknowledge the incoming meeting
+   * @returns {Promise}
+   * @memberof Meeting
+   */
+  acknowledge() {
+    return this.meetingRequest
+      .acknowledgeMeeting({
+        locusUrl: this.locusUrl,
+        deviceUrl: this.deviceUrl,
+        id: this.id
+      })
+      .then((response) => {
+        this.trigger(EVENT_TRIGGERS.MEETING_ALERTED, this);
+        return response;
+      });
+  }
+
+  /**
+   *
+   * Decline this meeting
+   * @param {String} reason
+   * @returns {Promise}
+   * @memberof Meeting
+   */
+  decline(reason = BUSY) {
+    return this.meetingRequest
+      .declineMeeting({
+        locusUrl: this.locusUrl,
+        deviceUrl: this.deviceUrl,
+        reason
+      })
+      .then((response) => this.spark.meetings.destroy(this, response));
```

Comment:
Do we want to destroy on a decline? I'm thinking of a case where I've been invited to a meeting but am on another call. I may want to decline to finish the current call and then hop in via obtp to the one I was invited in. I'm not sure if this is just an edge case though

Reply:
This is just for 1:1 call, so in this case we would want to. For space meeting, we would not want to destroy, it will go through a different flow.